test/ex_uid2_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule ExUid2Test do
```

Comment:
Are you going to write tests as part of this PR?

Reply:
No. I am working on tests in another branch to keep the size reasonable (but I _am_ writing tests I swear ;-) )